scripts/run-integration-tests

```diff
@@ -7,6 +7,7 @@ export TMPDIR=/tmp
 
 echo "build test contracts"
 cd ../integration_tests/contracts
+ln -s $(realpath "../../x/cronos/events/bindings/src") ./contracts/src
```

Comment:
I mean commit the symbolic link file into repo, so we don't need to add this line at all ;D

Reply:
why not, everyone need gen their own symlink

Reply:
Can we just create the file, do we have to create on the fly?

Reply:
You is right, change later